modules/installation-configuration-parameters.adoc

```diff
@@ -116,9 +118,6 @@ endif::[]
 ifeval::["{context}" == "installing-gcp-vpc"]
 :gcp:
 endif::[]
-ifeval::["{context}" == "installing-gcp-shared-vpc"]
```

Comment:
As it stands, you are continuing to delete GCP and IBM Cloud VPC content, plus what appears to be other updates. For example, you seem to be deleting content around bare metal content.

Reply:
It's completely ridiculous, 90% of these changes are not mine, including deleting GCP and IBM Cloud VPC. I'll save out my changes and revert this file.

Reply:
@mjpytlak this has been fixed